icu4c/source/test/intltest/rbbitst.cpp

```diff
@@ -1804,23 +1804,25 @@ class RegexRule : public SegmentationRule {
 //
 //---------------------------------------------------------------------------------------
 class RBBIMonkeyKind {
-public:
-    // Return a UVector of UnicodeSets, representing the character classes used
+  public:
```

Comment:
FYI / not blocking -- in ICU we normally don't indent labels but leave them aligned with the top level of their containing class or function definition